client/my-sites/sidebar-unified/static-data/jetpack-menu.js

```diff
@@ -0,0 +1,248 @@
+
+export default function jetpackMenu( { siteDomain } ) {
+	return [
+		{
+			icon: 'dashicons-chart-bar',
+			slug: 'stats',
+			title: translate( 'Stats' ),
+			type: 'menu-item',
+			url: `/stats/day/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-cart',
+			slug: 'upgrades',
+			title: translate( 'Upgrades' ),
+			type: 'menu-item',
+			url: `/plans/${ siteDomain }`,
+			children: [
+				{
+					parent: 'upgrades',
+					slug: 'upgrades',
+					title: translate( 'Plans' ),
+					type: 'submenu-item',
+					url: `/plans/${ siteDomain }`,
+				},
+				{
+					parent: 'upgrades',
+					slug: 'upgrades',
+					title: translate( 'Purchases' ),
+					type: 'submenu-item',
+					url: `/purchases/subscriptions/${ siteDomain }`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-post',
+			slug: 'edit-php',
+			title: translate( 'Posts' ),
+			type: 'menu-item',
+			url: `/posts/${ siteDomain }`,
+			children: [
+				{
+					parent: 'edit.php',
+					slug: 'edit-php',
+					title: translate( 'All Posts' ),
+					type: 'submenu-item',
+					url: `/posts/${ siteDomain }`,
+				},
+				{
+					parent: 'edit.php',
+					slug: 'post-new-php',
+					title: translate( 'Add New' ),
+					type: 'submenu-item',
+					url: `/post/${ siteDomain }`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-media',
+			slug: 'upload-php',
+			title: translate( 'Media' ),
+			type: 'menu-item',
+			url: `/media/${ siteDomain }`,
+		},
+		{
+			icon: 'dashicons-admin-page',
+			slug: 'edit-phppost_typepage',
+			title: translate( 'Pages' ),
+			type: 'menu-item',
+			url: `/pages/${ siteDomain }`,
+			children: [
+				{
+					parent: 'edit.php?post_type=page',
+					slug: 'edit-phppost_typepage',
+					title: translate( 'All Pages' ),
+					type: 'submenu-item',
+					url: `/pages/${ siteDomain }`,
+				},
+				{
+					parent: 'edit.php?post_type=page',
+					slug: 'post-new-phppost_typepage',
+					title: translate( 'Add New' ),
+					type: 'submenu-item',
+					url: `/page/${ siteDomain }`,
+				},
+			],
+		},
+		{
+			icon: 'dashicons-admin-comments',
+			slug: 'edit-comments-php',
+			title: translate( 'Comments' ),
+			type: 'menu-item',
+			url: `/comments/${ siteDomain }`,
```

Comment:
```suggestion
			url: `/comments/all/${ siteDomain }`,
```

https://github.com/Automattic/jetpack/blob/6cc6ec6b0243bb92767efd30a1214cf55e0cc2cb/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L348

Reply:
Very good catch!